src/security-council-mgmt/SecurityCouncilManager.sol

```diff
@@ -80,64 +108,60 @@ contract SecurityCouncilManager is Initializable, ArbitrumTimelock, ISecurityCou
         require(_newCohort.length == 6, "SecurityCouncilManager: invalid cohort length");
         // TODO: ensure no duplicates accross cohorts; this should be enforced in nomination process. If there are duplicates, this call will revert in the Gnosis safe contract
         address[] memory previousMembersCopy;
-        if (_cohort == Cohort.MARCH) {
-            previousMembersCopy = SecurityCouncilMgmtUtils.copyAddressArray(marchCohort);
-            marchCohort = _newCohort;
-        } else if (_cohort == Cohort.SEPTEMBER) {
-            previousMembersCopy = SecurityCouncilMgmtUtils.copyAddressArray(septemberCohort);
-            septemberCohort = _newCohort;
+        if (_cohort == Cohort.FIRST) {
+            previousMembersCopy = SecurityCouncilMgmtUtils.copyAddressArray(firstCohort);
```

Comment:
is `previousMembersCopy` not needed anymore?

Reply:
oh true